src/main/scala/io/citrine/theta/BenchmarkRegistry.scala

```diff
@@ -66,9 +68,9 @@ object BenchmarkRegistry {
   }
 
   /** Map from name to benchmark object */
-  val benchmarks: mutable.Map[String, Benchmark] = mutable.HashMap()
+  val benchmarks: mutable.Map[String, BenchmarkBuilder] = mutable.HashMap()
   register("RandomGeneration", RandomGenerationBenchmark)
-  register("STREAM", new StreamBenchmark())
+  register("STREAM", StreamBenchmark)
```

Comment:
Why ALL CAPS for stream vs Camel Case RandomGeneration

Reply:
It's STREAM's convention: https://www.cs.virginia.edu/stream/